app/com/github/pedrovgs/dotto/core/algebra/dotto.scala

```diff
@@ -0,0 +1,27 @@
+package com.github.pedrovgs.dotto.core.algebra
+
+import cats.free.Free
+import com.github.pedrovgs.dotto.core.types.{LedInteraction, MorseSymbol}
+
+/**
+  * Application algebra used to describe what our program does. Implemented using Cats as core library.
+  */
+object dotto {
+
+  type Message = String
+  type MorseSentence = Seq[Seq[MorseSymbol]]
```

Comment:
Avoid use of `Seq` in favor of `List`. `Seq` is by default a mutable collection as stated in the scala default Predef.